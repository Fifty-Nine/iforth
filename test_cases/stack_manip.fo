."noop abc " "abc\n" .s clear
."swap abc " "abc\n" swap .s clear
."dup  abc " "abc\n" dup .s clear
."over abc " "abc\n" over .s clear
."rot  abc " "abc\n" rot .s clear
."drop abc " "abc\n" drop .s clear
