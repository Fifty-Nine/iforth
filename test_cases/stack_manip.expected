noop abc abc
swap abc bac
dup  abc aabc
over abc babc
rot  abc cab
drop abc bc
